core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -111,21 +110,21 @@ object Kleisli extends KleisliInstances with KleisliFunctions with KleisliExplic
     }
 }
 
-sealed private[data] trait KleisliFunctions {
+private[data] sealed trait KleisliFunctions {
 
   def liftF[F[_], A, B](x: F[B]): Kleisli[F, A, B] =
     Kleisli(_ => x)
 
   /**
-   * Same as [[liftF]], but expressed as a FunctionK for use with mapK
-   * {{{
-   * scala> import cats._, data._, implicits._
-   * scala> val a: OptionT[Eval, Int] = 1.pure[OptionT[Eval, ?]]
-   * scala> val b: OptionT[Kleisli[Eval, String, ?], Int] = a.mapK(Kleisli.liftK)
-   * scala> b.value.run("").value
-   * res0: Option[Int] = Some(1)
-   * }}}
-   */
+    * Same as [[liftF]], but expressed as a FunctionK for use with mapK
+    * {{{
+    * scala> import cats._, data._, implicits._
+    * scala> val a: OptionT[Eval, Int] = 1.pure[OptionT[Eval, ?]]
+    * scala> val b: OptionT[Kleisli[Eval, String, ?], Int] = a.mapK(Kleisli.liftK)
+    * scala> b.value.run("").value
+    * res0: Option[Int] = Some(1)
+    * }}}
+    */
```

Comment:
This is IntelliJ autoformating, I assume?

Reply:
ugh. For some reason it didn't enable scalafmt in cats, it seems...